const.py

```
@@ -0,0 +1,2 @@
+VERSION = "v0.3.15"
```

Ruff (0.8.2)

Missing docstring in public module

(D100)